client/src/components/NavFooter.js

```diff
@@ -0,0 +1,70 @@
+import styled from 'styled-components/macro';
+import { NavLink } from 'react-router-dom';
+
+import HomeButton from '../images/navHome.svg';
+import HomeButtonActive from '../images/navHomeActive.svg';
+import ShelfButton from '../images/navShelves.svg';
+import ShelfButtonActive from '../images/navShelvesActive.svg';
+import BooksButton from '../images/navBooks.svg';
+import BooksButtonActive from '../images/navBooksActive.svg';
+
+export default function NavFooter({ isStatic }) {
+  return (
+    <NavWrapper isStatic={isStatic}>
+      <NavLink to="/home" activeClassName="active">
+        <NavHome role="img" aria-label="Home"></NavHome>
```

Comment:
I like your usage of `role="img"` with `aria-label="Home"`!

Reply:
On the other hand: why not use a real image element? It could be wrapped in a component that shows one or the other imageSrc. It's still fine as you do it here, I just wonder.